lib/formatters/default.js

```diff
@@ -14,12 +14,16 @@ class DefaultPrinter {
     let printOptions = {
       console: this.console,
       includeTodo: options.includeTodo,
+      isInteractive: process.stdout.isTTY,
+      outputFile: options.outputFile,
       quiet: options.quiet,
       updateTodo: options.updateTodo,
       verbose: options.verbose,
+      workingDirectory: options.workingDirectory,
```

Comment:
Since `Rule` has `workingDir` already as a public API, lets do:

```suggestion
      workingDir: options.workingDirectory,
```

Reply:
Done.